### PR TITLE
Remove HTMLAttrXRef and mentions from pt-BR

### PR DESCRIPTION
**Pull request by cw118**
Removes `{{HTMLAttrXRef}}` macros and its mentions in the docs for pt-BR to help with #5603.

### CODE REVIEW

File: files/pt-br/web/api/window/error_event/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ element.onerror = function(event) { ... }
...
+Quando um erro de sintaxe**(?)** ocorre em um script, carregado de uma [origem diferente](/pt-BR/docs/Web/Security/Same-origin_policy), os detalhes do erro de sintaxe não são reportados para previnir vazamento de informações (veja {{bug("363897")}}). Ao invés de exibir simplesmente **`"Script error." (erro de script)`**, este comportamento pode ser sobrescrito em alguns navegadores usando o atributo `[`crossorigin`](/pt-BR/docs/Web/HTML/Element/script#crossorigin)` no {{HTMLElement("script")}} e tendo o servidor enviado os cabeçalhos HTTP CORS apropriados. Uma solução alternativa é isolar o "Script error." e manipulá-lo sabendo que o detalhe do erro é visível somente no console do navegador e não acessível através do JavaScript.
```
Comment (josielrocha) [inline files/pt-br/web/api/window/error_event/index.md:46]: image
```suggestion
Quando um erro de sintaxe**(?)** ocorre em um script, carregado de uma [origem diferente](/pt-BR/docs/Web/Security/Same-origin_policy), os detalhes do erro de sintaxe não são reportados para previnir vazamento de informações (veja {{bug("363897")}}). Ao invés de exibir simplesmente **`"Script error." (erro de script)`**, este comportamento pode ser sobrescrito em alguns navegadores usando o atributo [crossorigin](/pt-BR/docs/Web/HTML/Element/script#crossorigin) no {{HTMLElement("script")}} e tendo o servidor enviado os cabeçalhos HTTP CORS apropriados. Uma solução alternativa é isolar o "Script error." e manipulá-lo sabendo que o detalhe do erro é visível somente no console do navegador e não acessível através do JavaScript.
```

**Comment by cw118** [comment]
Ah I missed that, thank you!

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Comment by josielrocha** [comment]
Thank you so much for your help @cw118!

**Pull request merged**